nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -288,3 +288,5 @@ In addition to numerous new and upgraded packages, this release has the followin
 - The option `services.prometheus.exporters.pihole.interval` does not exist anymore and has been removed.
 
 - `k3s` can now be configured with an EnvironmentFile for its systemd service, allowing secrets to be provided without ending up in the Nix Store.
+
+- `gitea` module options have been made more RFC042 conforming (i.e. more options are now in `settings`)
```

Comment:
I wouldn't expect an average NixOS admin who's not involved with upstream development at all to know what RFC42 is about (despite the nice number :p)

Hence I'd prefer to provide a few more details, i.e. that a bunch of options don't exist anymore, but can be set with `services.gitea.settings` and that the keys match the upstream config params now.